mac-setup-normal.sh

```diff
@@ -53,6 +53,9 @@ copy_ssh_key () {
   elif [ -e ~/.ssh/id_dsa ]
   then
     pbcopy < ~/.ssh/id_dsa.pub
+  elif [ -e ~/.ssh/id_ecdsa ]
+  then
+    pbcopy < ~/.ssh/id_ecdsa.pub
   else
```

Comment:
Do you want to put this first?  Like, if both rsa and ecdsa exist, which should we prefer?  Maybe it doesn't matter.

Reply:
Good idea. I updated it to put ecdsa first 👍